services/PostService.js

```diff
@@ -34,6 +46,11 @@ module.exports = class PostService {
             [Op.like]: id,
           },
         },
+        include: [
+          {
+            model: Comment,
+          },
+        ],
```

Comment:
when you do this the function is not all post anymore, and not find one post anymore. It is different query all together.